packages/validator/src/cli-validator/run-validator.js

```diff
@@ -261,6 +254,13 @@ const processInput = async function(program) {
       if (debug) {
         console.log(err.stack);
       }
+      // this check can be removed once we support spectral overrides
+      if (err.message.startsWith('Document must have some source assigned.')) {
+        console.log(
+          'This error likely occurred because Spectral `exceptions` are deprecated and `overrides` are not yet supported.\n' +
+            'Remove these fields from your Spectral config file to proceed.'
+        );
```

Comment:
Due to some bizarre behavior in Spectral, we can't support overrides yet. That will have to be a follow on change. This gives users a better error message than Spectral does